### PR TITLE
Add automated test for TGIS Standalone serving runtime (gRPC CLI)

### PR DESCRIPTION
**Pull request by bdattoma**
Testing of TGIS standalone serving runtime with single-model serving platform. Covering RHODS-13281.

Main Changes:
- New test suite added `ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422_model_serving_llm_other_runtimes.robot`
- Make keyword to query models a bit more generic: `ods_ci/tests/Resources/CLI/ModelServing/llm.resource`
- add a json file reporting the query formats for the runtimes: `ods_ci/tests/Resources/Files/llm/runtime_query_formats.json`
- update the ServingRuntime images: it's temporarily pointing to a specific digest, but will find a better logic to avoid overwriting it every release or just miss the change

My goal is to get to a point where we can easily write deploy/query test for a serving runtime with zero changes in the common keywords (or at least very minimal). Not sure if that'll be possible given the differences btw runtimes, but this is an attempt..

### CODE REVIEW

**Comment by bdattoma** [comment]
PR Validaition:
1. TGIS standalone CLI test: `rhods-ci-pr-test/2324`
2. Model Serving LLM Smoke suite for regression: _coming soon_
3. Model Serving UI Smoke suite for regression: _probably cannot run because UI tests are broken for 2.6 changes_

File: ods_ci/tests/Resources/Files/llm/serving_runtimes/tgis_servingruntime_grpc.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      image: quay.io/opendatahub/text-generation-inference:stable
+      command: ["text-generation-launcher"]
+      args:
+        - "--model-name=/mnt/models/"
+        - "--port=3000"
+        - "--grpc-port=8033"
+      env:
+        - name: TRANSFORMERS_CACHE
+          value: /tmp/transformers_cache
+      # resources: # configure as required
```
[the remark this answers is not in the document]
Comment (bdattoma) [reply ods_ci/tests/Resources/Files/llm/serving_runtimes/tgis_servingruntime_grpc.yaml:21]: I'm taking this configuration from https://github.com/opendatahub-io/caikit-tgis-serving/pull/186/files#diff-984479a2329d807e9c9daed95d2d296d56c5ae4723fad5770c8fde6f9271b2bf and thought to leave it as it is there

File: ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422_model_serving_llm_other_runtimes.robot [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+${FLAN_MODEL_S3_DIR}=    flan-t5-small/flan-t5-small-hf
+${FLAN_STORAGE_URI}=    s3://${S3.BUCKET_3.NAME}/${FLAN_MODEL_S3_DIR}/
+${TGIS_RUNTIME_NAME}=    tgis-runtime
+
+
+*** Test Cases ***
+Verify User Can Serve And Query A Model With TGIS-Standalone Runtime (gRPC)
+    [Documentation]    Basic tests for preparing, deploying and querying a LLM model
+    ...                using Kserve and TGIS standalone runtime
+    [Tags]    Sanity    Tier1    ODS-XYZ
```
[the remark this answers is not in the document]
Comment (bdattoma) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422_model_serving_llm_other_runtimes.robot:25]: yep, need to update it. thx

[the remark this answers is not in the document]
Comment (bdattoma) [reply ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/422_model_serving_llm_other_runtimes.robot]: done

**Review by FedeAlonso — DISMISSED** [review]

**Review by jstourac — APPROVED** [review]
Note: I have a very limited knowledge about this.

From the general point of view this LGTM (well, trailing spaces could be avoided next time :slightly_smiling_face: )

**Review by tarukumar — APPROVED** [review]

**Pull request merged**